config.yml

```diff
@@ -1,14 +1,15 @@
 # These track the column number of each player attribute in playerlist.csv file
 # Note that change in csv column order can be edited here.
 player_attribute_index:
-  telegram_username: 0
-  name: 1
-  gender_pref: 2
-  gender: 3
-  interests: 4
-  two_truths_one_lie: 5
-  introduction: 6
-  house_number: 7
-  cg_number: 8
-  year_of_study: 9
-  faculty: 10
\ No newline at end of file
+  telegram_username: 1
+  name: 0
+  room_number: 3
+  house_number: 2
+  faculty: 4
+  gender: 6
+  gender_pref: 5
+  year_of_study: 7
+  likes: 9
+  dislikes: 10
+  comments: 8
```

Comment:
Why is the ordering so wack haha

Reply:
I was matching the columns of the sample data csv file